run-ib-igprof

```diff
@@ -4,11 +4,16 @@ case $CMS_BOT_DIR in /*) ;; *) CMS_BOT_DIR=$(pwd)/${CMS_BOT_DIR} ;; esac
 WORKFLOWS=$1
 PROFILES=$2
 EVENTS=$3
+HPUWF="23434.21"
 if [ "X$EVENTS" = "X" ] ; then EVENTS=1000; fi
 for prof in ${PROFILES} ; do
   mkdir -p $WORKSPACE/igprof/${prof}
   cd $WORKSPACE/igprof/${prof}
   runTheMatrix.py $WORKFLOWS --ibeos --command "-n $EVENTS --profile $prof" 2>&1 | tee runTheMatrix.log
+  if [[ $WORKFLOWS == *"$HPUWF" ]]; then
```

Comment:
@gartung , this will match any workflow which ends with $HPUWF . Is this what we want here?

Reply:
Yes. The jenkins job has WORKFLOWS="-l 23434.21"